New hallucinations

AnturK opened the pull request:
New severe hallucinations that can happen instead of punchghosts
- Alien Hunter
- Plasma flood
- Singularity
- Fight sounds
- Clown-visions
- Fake bolted doors
- Fake speech/radio messages.

AnturK committed:
New hallucinations

RemieRichards commented on code/modules/flufftext/Hallucination.dm:
```diff
@@ -176,30 +176,296 @@ Gunshots/explosions/opening doors/less rare audio (done)
+
+/obj/effect/hallucination/fake_flood/process()
+	if(next_expand <= world.time)
+		radius += 1
```
radius++

AnturK replied:
This is probably going to be rewritten to actually flood anyway but sure.

SconesC commented:
Can you add more sounds to the random sound list?
- Welding
- Screwdriver
- Esword on/off
- Canister burst

Iamgoofball commented on code/modules/flufftext/Hallucination.dm:
```diff
@@ -176,30 +176,296 @@ Gunshots/explosions/opening doors/less rare audio (done)
+			for(var/i=0,i<hits,i++)
+				target << sound('sound/weapons/blade1.ogg',,0,1,0,25)
+				sleep(rand(1,5))
+			target << sound(get_sfx("bodyfall"),0,1,0,25)
```
keep doing hit sounds after the body falls a few times, a proper traitor finishes the job instead of relying on crit

Iamgoofball commented on code/modules/flufftext/Hallucination.dm:
```diff
@@ -176,30 +176,296 @@ Gunshots/explosions/opening doors/less rare audio (done)
+			for(var/i=0,i<hits,i++)
+				target << sound('sound/weapons/Laser.ogg',0,1,0,25)
+				sleep(rand(1,5))
+			target << sound(get_sfx("bodyfall"),0,1,0,25)
```
do some punching sounds after this, or thwacking sounds

AnturK committed:
Adds fake bolts hallucination

SconesC commented:
Could also have a battle sound of stunprod + toolboxing

AnturK committed:
Adds Cablecuff/Zipties sound Hallucination
Adds some other sounds
Adds Bolted doors hallucination
Adds Speech/Radio hallucination

AnturK committed:
Small update

AnturK committed:
Whispers Hallucination
Minor Adjustments
Scaling hallucinations

AnturK committed:
Minor progress

AnturK committed:
Minor messages

AnturK commented:
This is possibly merge ready. Some hallucinations could use a bit of a realism(namely singulo) but otherwise all works. No changelog because spooky.

phil235 commented:
You need to remove var/list/atom/hallucinations from living_defines.dm since it's unused now.

phil235 commented on code/modules/flufftext/Hallucination.dm:
```diff
@@ -356,3 +513,251 @@ var/list/non_fakeattack_weapons = list(/obj/item/weapon/gun/projectile, /obj/ite
+		"<span class='notice'>You hear something squeezing through the ducts...</span>", \
+		"<span class='notice'>You hear a distant scream.</span>", \
+		"<span class='notice'>Your </span>", \
+		"<B>[target]</B> sneezes.", \
```
This message has been split in two somehow...

AnturK committed:
Minor fixes
